Cargo.toml

```diff
@@ -120,7 +120,7 @@ etcd-client = { version = "0.13" }
 fst = "0.4.7"
 futures = "0.3"
 futures-util = "0.3"
-greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "973f49cde88a582fb65755cc572ebcf6fb93ccf7" }
+greptime-proto = { git = "https://github.com/realtaobo/greptime-proto.git", rev = "8671bd04ebca7a45880d10a3d4a1102f4fd5d8f1" }
```

Comment:
Modify the `greptime-proto` before merging. (Waiting for https://github.com/GreptimeTeam/greptime-proto/pull/189)